install/kubernetes/helm/istio/charts/istiocoredns/templates/configmap.yaml

```diff
@@ -13,11 +13,17 @@ data:
     .:53 {
           errors
           health
+          {{ if eq -1 (semver  .Values.coreDNSTag  | (semver "1.4.0").Compare) }}
```

Comment:
Nit: we can define a variable above for "1.4.0" version
```
$version := semver "1.4.0"
```

Reply:
What would be the benefit since we use this variable only at this place and it's more or less a constant ?

Reply:
Never mind, just make it more readable.